salt/metalk8s/volumes/unprepared/init.sls

```diff
@@ -1,19 +1,27 @@
 {%- set volumes = pillar.metalk8s.volumes %}
-{%- set volume  = pillar.get('volume', '') %}
+{%- set volume_name  = pillar.get('volume', '') %}
+{%- set to_remove = volumes.get(volume_name) %}
 
+{%- if to_remove is not none %}
+  {%- if 'sparseLoopDevice' in to_remove.spec %}
+Disable systemd provisioning of loop device for {{ volume_name }}:
+  service.disabled:
```

Comment:
Will this also 'stop' the service?

Reply:
Good catch. I didn't realize it wasn't stopped, because I left the cleanup logic in the module :confused: